src/libpostalParser.js

```diff
@@ -0,0 +1,44 @@
+var logger = require('pelias-logger').get('text-analyzer');
+
+// mapping object from libpostal fields to pelias fields
+var field_mapping = {
+  category:       'category',
+  house:          'query',
+  house_number:   'number',
+  road:           'street',
+  suburb:         'neighbourhood',
+  city_district:  'borough',
+  city:           'city',
+  state_district: 'county',
+  state:          'state',
+  postcode:       'postalcode',
+  country:        'country'
+};
+
+// wrapper for libpostal that injects the actual parse function for easier
+// testing purposes
+module.exports.create = function create(parse_address) {
```

Comment:
would be really helpful to add a doc block here and explain what `parse_address` is for and what it's expected type / structure / interface should be.